<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UC_Employees</class>
 <widget class="QFrame" name="UC_Employees">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="Frame_Employees">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>860</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="Frame_Employees_Reports">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_136">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>إلى</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dtTo">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="displayFormat">
      <string>M/d/yyyy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_action_search">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>بحث</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_print">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>طباعه</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_132">
     <property name="geometry">
      <rect>
       <x>944</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الموظف</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dtFrom">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="displayFormat">
      <string>M/d/yyyy</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxEmp">
     <property name="geometry">
      <rect>
       <x>659</x>
       <y>10</y>
       <width>241</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tblEmpActions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>108</y>
       <width>1021</width>
       <height>691</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>الرقم التعريفى</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الإسم</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>نوع النشاط</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>التاريخ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>التفاصيل</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="cbxActionTypes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>241</width>
       <height>39</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>- - - - - - -</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>   اضافة</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>   تعديل</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>   حذف</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_135">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الفتره من</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_133">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>  نوع الحدث</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Frame_Employees_Info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>البريد الإلكترونى</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmpName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove_employee">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>159</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>تاريخ بداية العمل </string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tbl_emp_info">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>1024</width>
       <height>522</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>رقم التعريف</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الإسم</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>رقم الهاتف</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>البريد الالكترونى</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>العنوان</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>القسم</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الراتب</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>تاريخ بداية العمل</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>150</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>العنوان</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>90</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>رقم الهاتف</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>210</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>القسم</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_edit_employee">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>210</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>تعديل</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dtEmpStartWork">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="displayFormat">
      <string>M/d/yyyy</string>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>5</day>
      </date>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmpID">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmpSalary">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>210</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmpPhone">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>80</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_274">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>210</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الراتب</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxEmpDep">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>210</y>
       <width>121</width>
       <height>39</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>قسم1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>قسم2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>قسم3</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtEmpAddress">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>150</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>رقم التعريف</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>ألاسم</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_employee">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>إضافة</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmpMail">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_employees_frame">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>بيانات الموظفين</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_employees_archieve_frame">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>نشاطات الموظفين</string>
    </property>
   </widget>
   <zorder>Frame_Employees_Info</zorder>
   <zorder>btn_employees_frame</zorder>
   <zorder>btn_employees_archieve_frame</zorder>
   <zorder>Frame_Employees_Reports</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
